frontend/src/App.js

```diff
@@ -29,7 +33,7 @@ function App() {
             <InDepthDetailView />
           </Route>
           <Route path={'/'}>
-            <Redirect to={'/overview'} />
+            <Redirect to={'/overviewlist'} />
```

Comment:
If you want `/overviewlist` to be your default route you can define it like this
```
<Route path={['/', '/overviewlist']} component={OverViewList} />
```